src/commands/internal/create-changelog.ts

```diff
@@ -40,6 +40,10 @@ export async function run(...args): Promise<boolean> {
 }
 
 export async function getChangelogText(startRef: string): Promise<string> {
+  if (!startRef) {
+    return '';
+  }
+
   const apiResponse = await getCommitFromApi(startRef);
 
   if (apiResponse.commit === undefined) {
```

Comment:
Wenn auch nicht Teil des PR (Pfadfinder-Prinzip):

```suggestion
  if (apiResponse.commit == null) {
```